spec/rails_helper.rb

```diff
@@ -19,4 +19,5 @@
   config.filter_rails_from_backtrace!
   config.include FactoryBot::Syntax::Methods
   config.include(Shoulda::Matchers::ActiveRecord, type: :model)
+  config.include ResponseJSON
```

Comment:
Please be consistent. Poetry mode or not. (parentheses or no parentheses..)

Reply:
I suppose the parantheses are necessary for line 21 due to the two arguments. Anyway you might want to update the teachable material then :)
(end of page https://learn.craftacademy.co/courses/676568/lectures/12064426)
![Screenshot 2020-02-05 at 10 54 03](https://user-images.githubusercontent.com/56539700/73830994-e0191d00-4805-11ea-97ea-1bc6fc7581a2.png)

Reply:
parentheses added